Sources/SwiftSyntax/Utils.swift

```diff
@@ -102,3 +102,49 @@ extension RawUnexpectedNodesSyntax {
     self.init(raw: raw)
   }
 }
+
+extension String {
+  func trimmingCharacters(in charactersToTrim: any BidirectionalCollection<Character>) -> Substring {
```

Comment:
Based on my comments above, I’m not sure we ever want to trim all characters from a collection. If so, I’m not sure if these extensions are needed 😉